hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -87,6 +88,23 @@ public XceiverClientFactory getXceiverClientFactory() {
     return blockOutputStreamEntryPool.getLocationInfoList();
   }
 
+  @VisibleForTesting
```

Comment:
Looks like this used only for tests?
If so, why can't we use getStreamEntries API and build the required info in tests?

Reply:
Oh, sure we could put this into the test class, thanks~